schema/crdb/lookup-bgp-config-indexes/up01.sql

```diff
@@ -0,0 +1,4 @@
+CREATE INDEX IF NOT EXISTS lookup_bgp_config_by_bgp_announce_set_id ON omicron.public.bgp_config (
+    bgp_announce_set_id
+) WHERE
+    time_deleted IS NULL;
```

Comment:
Double check if this is needed for schema/crdb/lookup-bgp-config-indexes/up02.sql

Reply:
Same comment as above, yes this is needed.

Reply:
See follow up comment above.